src/pcapi/core/bookings/repository.py

```diff
@@ -424,6 +425,7 @@ def _serialize_booking_recap(booking: AbstractKeyedTuple) -> BookingRecap:
         "offer_name": booking.offerName,
         "offerer_name": booking.offererName,
         "beneficiary_email": booking.beneficiaryEmail,
+        "beneficiary_phonenumber": booking.beneficiaryPhoneNumber,
```

Comment:
```suggestion
        "beneficiary_phone_number": booking.beneficiaryPhoneNumber,
```

Pour coller avec la casse du camelCase